src/utils/simulateNonSkillHelper.ts

```diff
@@ -0,0 +1,117 @@
+import * as vscode from 'vscode';
```

Comment:
This name seems ambiguous to me. What are we expecting this file to contain? `replay` specific utility helpers or any grab bag of helpers for simulator? And why is it `NonSkill` ?

Reply:
This name represents not skill management methods. Yeah, it's better to rename it to simulateReplayHelper, and create new files for other new features :)